packages/circuit-ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@sumup/circuit-ui",
-  "version": "3.0.0-next.0",
+  "version": "3.0.0-next.1",
```

Comment:
🎉 this works

@connor-baer should I also update the versions in the dependencies and peer dependencies then?

Reply:
If you update the package versions, changesets should figure out the dependencies automatically.

Reply:
It didn't 🤔 I updated them on `next`